### PR TITLE
[CP-1899][E2E][Contacts] Add to favourites

### PR DESCRIPTION
**Pull request by rajmundex**
Jira: [CP-1899]

**Description**
added to existing test file new-contact

<details>
<summary><b>Screenshots</b></summary>
// put images here

image

</details>

[CP-1899]: https://appnroll.atlassian.net/browse/CP-1899?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/e2e/src/test-data/contacts-only-test-data.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+export const contacts: Array<{
+  firstName: string
+  lastName: string
+  primaryNumber: string
+  secondaryNumber: string
+  addressFirstLine: string
+  addressSecondLine: string
+  addToFavourites: boolean
+}> = [
```
Comment (patryk-sierzega) [inline packages/e2e/src/test-data/contacts-only-test-data.ts:9]: u can extract type of this Array to a type or interface :)

**Review by patryk-sierzega — APPROVED** [review]

File: packages/e2e/src/specs/contacts/new-contact.e2e.ts [hunk 2]
```diff
@@ -3,123 +3,302 @@
...
+    if (contactToAdd.addToFavourites) {
+      await contactsPage.addToFavouritessCheckbox.click()
+    }
+    await contactsPage.clickSaveContactButton()
+    await browser.pause(2000)
+    const storedContacts = await contactsPage.listOfContacts
+    expect(storedContacts.length).toBeGreaterThan(0)
+    const ContactData = await storedContacts[0].getText()
+    expect(ContactData).toHaveText(
+      `${contacts[0].firstName} ${contacts[0].lastName}`
```
Comment (lkowalczyk87) [inline packages/e2e/src/specs/contacts/new-contact.e2e.ts:168]: I think you want to use const `contactToAdd`

File: packages/e2e/src/specs/contacts/new-contact.e2e.ts [hunk 3]
```diff
@@ -3,123 +3,302 @@
...
+    )
+
+    const phoneNumber = await contactsPage.phoneNumberOnContactList
+    expect(phoneNumber).toHaveTextContaining([
+      contacts[0].primaryNumber,
```
Comment (lkowalczyk87) [inline packages/e2e/src/specs/contacts/new-contact.e2e.ts:173]: I think you want to use const `contactToAdd`

File: packages/e2e/src/specs/contacts/new-contact.e2e.ts [hunk 4]
```diff
@@ -3,123 +3,302 @@
...
-    await ContactsPage.insertTextToFirstNameInput("Henryk")
-    await ContactsPage.insertTextToLastNameInput("Zaskroniec")
-    await ContactsPage.insertTextToPrimaryPhoneNumberInput("602900000")
-    await ContactsPage.insertTextToSecondaryPhoneNumberInput("510100100")
-    await ContactsPage.insertTextToAddressFirstLineInput(
-      "Al. Ujazdowskie 100/19"
+    await contactsPage.insertTextToFirstNameInput(contacts[1].firstName)
+    await contactsPage.insertTextToLastNameInput(contacts[1].lastName)
+    await contactsPage.insertTextToPrimaryPhoneNumberInput(
+      contacts[1].primaryNumber
```
Comment (lkowalczyk87) [inline packages/e2e/src/specs/contacts/new-contact.e2e.ts:30]: You can declare `const contactToAdd = contacts[1]` like you did in test below. A similar approach can be used for `storedContacts[0]`, `contacts[2]` etc.

**Review by OskarMichalkiewicz — APPROVED** [review]

File: packages/e2e/src/specs/contacts/new-contact.e2e.ts (hunk 4 above)
Comment (rajmundex) [reply packages/e2e/src/specs/contacts/new-contact.e2e.ts]: https://github.com/mudita/mudita-center/pull/1201/commits/08252851ec674568c6c4262bcebf0327ead77165

File: packages/e2e/src/specs/contacts/new-contact.e2e.ts (hunk 3 above)
Comment (rajmundex) [reply packages/e2e/src/specs/contacts/new-contact.e2e.ts]: https://github.com/mudita/mudita-center/pull/1201/commits/08252851ec674568c6c4262bcebf0327ead77165

File: packages/e2e/src/specs/contacts/new-contact.e2e.ts (hunk 2 above)
Comment (rajmundex) [reply packages/e2e/src/specs/contacts/new-contact.e2e.ts]: https://github.com/mudita/mudita-center/pull/1201/commits/08252851ec674568c6c4262bcebf0327ead77165

File: packages/e2e/src/test-data/contacts-only-test-data.ts (hunk 1 above)
Comment (rajmundex) [reply packages/e2e/src/test-data/contacts-only-test-data.ts]: https://github.com/mudita/mudita-center/pull/1201/commits/146739676ea20ca1dc855808c791cd6b3efc540c

**Review by patryk-sierzega — APPROVED** [review]

**Review by lkowalczyk87 — APPROVED** [review]

**Pull request merged**